.github/actions/setup/action.yml

```diff
@@ -70,6 +70,10 @@ runs:
         version: ${{ env.SOLANA_VERSION }}
         cache: true
 
+    - name: Install 'cargo-hack'
```

Comment:
Should we add an if statement here to ensure we only install this if `toolchain` is set to `lint`?

Reply:
Whichever toolchain they need. But otherwise JS CI steps will end up installing this when they don't need to.

Reply:
Switch to use `lint` in all scripts related to linting.